pkg/networks/commands.go

```diff
@@ -22,6 +24,36 @@ func (config *NetworksConfig) Check(name string) error {
 	return fmt.Errorf("network %q is not defined", name)
 }
 
+func (config *NetworksConfig) CheckDaemon(daemon string) (bool, error) {
```

Comment:
I found the name confusing at first; maybe this should be `IsDaemonInstalled` or `LocateDaemon` (but in the latter case I would expect the function to return the path to the daemon).

Reply:
Renamed to `IsDaemonInstalled`